java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -2743,6 +2745,19 @@ public static Table fromPackedTable(ByteBuffer metadata, DeviceMemoryBuffer data
     return result;
   }
 
+  /**
+   * Gather `n` samples from table randomly
+   * The output is not same with CPU Sample exec, but this is faster.
+   *
+   * @param n
```

Comment:
What does `n` mean ? It would be better to give a meaningful name, plus a short explaination.